cmd/create/machinepool/options_test.go

```diff
@@ -0,0 +1,59 @@
+package machinepool
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	mpOpts "github.com/openshift/rosa/pkg/options/machinepool"
+	"github.com/openshift/rosa/pkg/reporter"
+)
+
+var _ = Describe("CreateMachinepoolOptions", func() {
```

Comment:
@den-rgb Same comment here..really nice readability and conciseness on these tests. Very nice!

Reply:
Thank you :)